documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-configuration-database-history-properties.adoc

```diff
@@ -0,0 +1,78 @@
+{prodname} provides a set of `database.history.*` properties that control how the connector processes events that it reads from the schema history topic.
+These `database.history` properties specify how the {context} connector processes the event data that it reads from the database history.
+
+The following table describes the `database.history` properties for configuring the {prodname} {context} connector.
```

Comment:
Thanks for catching that. You're suggesting that we create a new variable for the connector name/context, along the lines e.g., of `{connector-name-context}`? That would is fine with me, but I'm not sure whether we need the extra overhead, unless there are lots of instances where it would provide added value over the simpler solution, which in this case means removing the connector name, so that the sentence reads _"... specify how the connector processes ..."_ WDYT?

Reply:
Yes, we can do that, too of course. In fact, it is what I will do now right away, so to get this one merged finally. You could have done yourself, you're the docs lead and should decide such matters :)